templates/batch-llm/README.ipynb

```diff
@@ -80,16 +83,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Set the Hugging Face token. Replace the following with your token.\n",
-    "HF_TOKEN = \"<REPLACE_WITH_YOUR_HUGGING_FACE_USER_TOKEN>\"\n",
-    "# Set to the model that you wish to use from the preceding list.\n",
-    "# Note that using the Llama models will require a Hugging Face token to be set.\n",
-    "HF_MODEL = \"mistralai/Mistral-7B-Instruct-v0.1\"\n",
+    "# Set to the name of the Hugging Face model that you wish to use from the preceding list.\n",
+    "# Note that using the Llama models will prompt you to set your Hugging Face user token.\n",
+    "HF_MODEL = \"meta-llama/Llama-2-7b-chat-hf\"\n",
```

Comment:
why we switch the model from Mistral to llama? This would require users to provide tokens, and harder for them to start.

Reply:
ah good call, let me change this back to mistral, thanks